luxtronik/parameters.py

```diff
@@ -893,9 +893,9 @@ def __init__(self, safe=True):
             849: Unknown("ID_Ba_Hz_MK3_saved"),
             850: Hours("ID_Einst_Kuhl_Zeit_Ein_akt", True),
             851: Hours("ID_Einst_Kuhl_Zeit_Aus_akt", True),
-            852: Unknown("ID_Waermemenge_Seit"),
+            852: Energy("ID_Waermemenge_Seit"),
```

Comment:
Do we know what `Seit` refers to here? Are we sure this is an Energy and not some Timestamp value?

Reply:
Yes. This seems to be the energy since the time stamp from parameter 880.